### ISSUE TITLE
Not an issue, an idea  single 2x audio interface for multiplayer.

### ISSUE THREAD

**Issue by NathanMast** [Maxx53/NoCableLauncher#11]
Using a 2x2 Audiobox interface, input1 is on the left channel, while input2 is on the right channel, but they both use a single recording device, if someone could figure out a way to emulate each side of the audio as an individual device, we could then use a single 2 input usb interface for multiplayer and that would be pretty awesome.

I am not sure if the libraries used have this ability or not but it would be a pretty neat quality of life update.

**Comment by Maxx53** [comment]
Yeah, thought about It. Same sutuatuon, my device also has 2 inputs. Technically, It's one recording device with stereo signal, 2 inputs at 2 channels. Rocksmith uses left channel of your device by default.
Only way is a virtual audio device (just like Virtual Audio Cable), with redirected signal from 2nd input of your device. I suppose, will be a huge delay in this case.
